buildlib/az-network-corrupter.sh

```diff
@@ -25,10 +27,14 @@ sleep ${initial_delay}
 
 for i in $(seq 1 ${cycles}); do
     echo "#$i Put it down! And sleep ${downtime}"
-    ${manager_script} --one -d ${manager_script_dir}/hosts --host $(hostname) -a off -i ${interface}
+    for interface in ${interfaces} ; do
+        ${manager_script} --one -d ${manager_script_dir}/hosts --host $(hostname) -a off -i ${interface}
+    done
```

Comment:
i think shutting off one random NIC should be enough

Reply:
here I do it
--one - choose one random interface
-a off - turn off